src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.ThreadCounts.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace System.Threading
+{
+    internal static partial class ClrThreadPool
+    {
+        /// <summary>
+        /// Tracks information on the number of threads we want/have in different states in our thread pool.
+        /// </summary>
+        /// <remarks>The size parameter ensures that we are always on our own cache line.</remarks>
+        [StructLayout(LayoutKind.Explicit, Size = 64)]
```

Comment:
This is not completely true, the size does not guarantee that the struct will be aligned to 64 bytes. The size specification is not necessary here since the static field of this type (where it matters) is already sufficiently separated from other memory.